### PR TITLE
Update documentation for Marker

### PR DESCRIPTION
**Pull request by sohibul**
Did you mean display the marker?

### CODE REVIEW

**Comment by alex3165** [comment]
@sohibul There is `Popup` on the following 2 lines as well, could you modify it as part of this PR ?

**Comment by sohibul** [comment]
Is [this](https://github.com/sohibul/react-mapbox-gl/commit/89022c0e2d7499edc97ad49f1c4ccbef502b5283) what you mean?

File: docs/API.md [hunk 1]
```diff
@@ -299,7 +299,7 @@ import { Marker } from "react-mapbox-gl";
 ```
 
 ### Properties
-- **coordinates** *(required)*: `Array<Number>` Display the popup at the given position.
+- **coordinates** *(required)*: `Array<Number>` Display the marker at the given position.
 - **anchor**: `String` Same as Popup's anchor property.
```
Comment (alex3165) [inline docs/API.md]: Here it is `Popup` and should be `Marker`

File: docs/API.md [hunk 2]
```diff
@@ -299,7 +299,7 @@ import { Marker } from "react-mapbox-gl";
...
 - **offset**: `String` Same as Popup's offset property.
```
Comment (alex3165) [inline docs/API.md]: Same here

**Comment by alex3165** [comment]
@sohibul See my comments

**Comment by sohibul** [comment]
sorry, i missed it

**Pull request merged**